typescript/.changeset/tame-tomatoes-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@goat-sdk/wallet-chromia": minor
```

Comment:
Seems more like a patch! You can simply update this and change `minor` for `patch`